src/CloseButton.js

```diff
@@ -10,14 +12,13 @@ class CloseButton extends React.Component {
   render() {
     const { label, onClick } = this.props;
     return (
-      <button
-        type="button"
+      <Button
```

Comment:
This won't work. Close buttons aren't implemented in terms of Buttons they are a separate thing. It's odd I know but that's how the styling works